koku/api/settings/tags/mapping/view.py

```diff
@@ -44,36 +40,26 @@ def get_annotated_queryset(self):
         return self.get_queryset().annotate(cost_model_id=Case(*when_conditions, output_field=UUIDField()))
 
 
-class SettingsTagMappingFilter(SettingsFilter):
-    source_type = django_filters.CharFilter(field_name="parent__provider_type", method="filter_by_source_type")
-    parent = django_filters.CharFilter(field_name="parent__key", lookup_expr="icontains")
-    child = django_filters.CharFilter(field_name="child__key", lookup_expr="icontains")
+class SettingsTagMappingFilter(TagMappingFilters):
+    source_type = CharFilter(field_name="parent__provider_type", method="filter_by_source_type")
+    parent = CharFilter(field_name="parent__key", method="filter_by_key")
+    child = CharFilter(field_name="child__key", method="filter_by_key")
```

Comment:
It seems like `SettingsFilter.filter_queryset` should be doing the work of parsing the input and returning the values. I think you just need to change these to `NonValidatedMultipleChoiceFilter` without all the other changes in this PR.

Reply:
Great suggestion Sam